### ISSUE TITLE
docker-compose up fails

### ISSUE THREAD

**Issue by ronreiter** [saasforge/open-source-saas-boilerpate#7]
Error: Cannot find module '/app/webpack.dev.js'

Happens because webpack.dev.js is not copied before the npm command. Need to fix the dockerfile.

**Comment by saasforge** [comment]
Hi, thanks for reporting this issue. Let me try to figure out how it can be fixed.

**Comment by saasforge** [comment]
@mandrews can you please look at this issue? Thanks!

**Comment by ericvanular** [comment]
@saasforge any idea how to fix this? Trying to use docker compose currently but stuck here

**Comment by saasforge** [comment]
@ericvanular Can you please describe step-by-step your problem? Thanks!

**Comment by ericvanular** [comment]
@saasforge same issue as @ronreiter 

I run docker-compose up and it throws an error with --> Error: Cannot find module '/app/webpack.dev.js'

**Comment by saasforge** [comment]
I'm not very familiar with docker but I think it's because there is no such folder _app_. What is /app folder at all? If it's a root one, where do you define it? If it's defined by default, you probably should change it to just / Can you please try?

**Comment by ericvanular** [comment]
Thanks @saasforge, I'm not super familiar with Docker either but the error is in Step 7/10: (RUN npm install --global npm && npm update && npm install && npm run dev)

I'll try to change the workdir name and report back

**Comment by ericvanular** [comment]
Was able to get the Docker image to build by switching the order of Step 7 and 8

COPY . /app

should be before 

RUN npm install --global npm && \
    npm update && \
    npm install && \
    npm run dev

**Comment by saasforge** [comment]
@ericvanular Thanks a lot! Do you want to make update to the dockfile or you want me to do that?

**Comment by ericvanular** [comment]
No worries, I can make a PR in a bit

**Comment by saasforge** [comment]
Cool, thanks!

**Comment by azziz-errime** [comment]
Hi there,

The suggested change doesn't appear in the code. Is it normal?

**Comment by saasforge** [comment]
No, it's not normal, it's just never been changed. Please feel free to create PR.

**Comment by saasforge** [comment]
Fixed in 87d795580748d942ab43ea23e2462cbf97b3f2b5
Please try and let us know if there is still an issue.
